pkg/sql/schemachanger/scbuild/testdata/alter_table_add_column

```diff
@@ -20,7 +20,7 @@ ALTER TABLE defaultdb.foo ADD COLUMN j INT NOT NULL DEFAULT 123
 - [[IndexColumn:{DescID: 104, ColumnID: 1, IndexID: 1}, ABSENT], PUBLIC]
   {columnId: 1, indexId: 1, tableId: 104}
 - [[PrimaryIndex:{DescID: 104, IndexID: 1, ConstraintID: 1}, ABSENT], PUBLIC]
-  {constraintId: 1, indexId: 1, isUnique: true, tableId: 104}
+  {constraintId: 1, geoConfig: {}, indexId: 1, isUnique: true, tableId: 104}
```

Comment:
If it's not too much trouble, I think it's worth eliding these empty structs from the test output somehow. This could be achieved by embedding the geoConfig struct in the index element, this way `scviz.RewriteEmbeddedIntoParent` would catch it. Perhaps there's a better way, I don't know.